CODEOWNERS

```diff
@@ -1,16 +1,20 @@
-# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# All
+*                              @jmeridth
 
-# Other and new charts
-/charts/ @oliverbaehler
+# Github files
+.github/                       @mkilchhofer @jmeridth
+
+# All charts
+/charts/                       @jmeridth
```

Comment:
That will not work. Github does not merge the OWNERS with the more granular rules :(

Already tried that when I started contributing here: 
https://github.com/argoproj/argo-helm/pull/1238